keywords/LiteServiOS.py

```diff
@@ -79,25 +204,56 @@ def start(self, logfile_name):
         4. Return the url of the running LiteServ
         """
 
-        if self.storage_engine != "SQLite":
-            raise NotImplementedError("Need to make sure to support other storage types")
+        data = {}
+        encryption_enabled = False
+        device = "iPhone-7-Plus"
+
+        package_name = "LiteServ-iOS.app"
+        app_dir = "LiteServ-iOS"
+
+        if self.storage_engine == "SQLCipher":
+            package_name = "LiteServ-iOS-SQLCipher.app"
+            app_dir = "LiteServ-iOS-SQLCipher"
+
+        app_path = "{}/{}/{}".format(BINARY_DIR, app_dir, package_name)
+
+        output = subprocess.check_output([
+            "ios-sim", "--devicetypeid", device, "--log", logfile_name, "launch", app_path, "--exit"
+        ])
+        log_info(output)
+
+        if self.storage_engine == "SQLite" or self.storage_engine == "SQLCipher":
+            data["storage"] = "SQLite"
+        elif self.storage_engine == "ForestDB" or self.storage_engine == "ForestDB+Encryption":
+            data["storage"] = "ForestDB"
+
+        if self.storage_engine == "ForestDB+Encryption" or self.storage_engine == "SQLCipher":
+            encryption_enabled = True
 
         self._verify_not_running()
 
         if self.port == 59850:
             raise LiteServError("On iOS, port 59850 is reserved for the admin port")
 
-        liteserv_admin_url = "http://{}:59850".format(self.host)
-        log_info("Starting LiteServ: {}".format(liteserv_admin_url))
+        data["port"] = int(self.port)
 
-        data = {
-            "port": int(self.port)
-        }
+        if encryption_enabled:
+            log_info("Encryption enabled ...")
 
+            db_flags = []
+            for db_name in REGISTERED_CLIENT_DBS:
+                db_flags.append("{}:pass".format(db_name))
+            db_flags = ",".join(db_flags)
+
+            log_info("Running with db_flags: {}".format(db_flags))
+            data["dbpasswords"] = db_flags
+
+        self._wait_until_reachable(port=59850)
+        liteserv_admin_url = "http://{}:59850".format(self.host)
```

Comment:
I thought that this was specific to iOS, I don't see such a usage anywhere in LiteServAndroid.py.